packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -142,9 +159,16 @@ export const Table = memo((props: Props) => {
     gotoPage,
     setPageSize,
     pageOptions,
+    toggleAllRowsExpanded,
   } = useTable(options, useFilters, useSortBy, useAbsoluteLayout, useResizeColumns, useExpanded, usePagination);
 
   const extendedState = state as GrafanaTableState;
+  toggleAllRowsExpandedRef.current = toggleAllRowsExpanded;
+
+  const expandedRowsRepr = JSON.stringify(Object.keys(state.expanded));
+  useEffect(() => {
+    listRef.current?.resetAfterIndex(0);
```

Comment:
I assume that this means, every time we change what is expanded, we recompute the variable sized list so the heights are correct?